src/Region.cpp

```diff
@@ -3,7 +3,6 @@
 #include <BWAPI/Regionset.h>
```

Comment:
Don't keep unnecessary includes, as they will be included from IteratorImpl -> Cast -> BWAPI.h